src/cirrus/build.py

```diff
@@ -45,6 +67,13 @@ def main():
         'venv',
         'bin',
         'virtualenv')
+
+    # remove existing virtual env if building clean
+    if opts.clean is True and os.path.exists(venv_path):
+        cmd = "rm -r {0}".format(venv_path)
```

Comment:
A good thing to keep in mind in general; the scope of cirrus is probably for *nix flavored OS's. Pragmatically this is OS X and Linux.